src/pages/api/auth-connect/set-password.ts

```diff
@@ -35,6 +45,8 @@ const handler: Handler = async (request, response) => {
         })
```

Comment:
Isn't there a span attribute missing in the `.then` case here ?

Reply:
Nice catch!